runtime/tutor/tutor1.zh.utf-8

```diff
@@ -393,7 +393,7 @@
 
   4. 更改類命令的格式是︰
 
-	 [number]   c	object	      或者	c   [number]   object
+	 [number]   c	object	      或	c   [number]   object
```

Comment:
You are right, I do not know whether it is right or not. But I didn't translate either. This file is created automatically from `tutor1.zh.big5`.

Reply:
ah okay.

Reply:
Hmm, that's weird.
In MS Windows powershell converts this file with these changes. But iconv versions 1.16 and 1.17 cannot convert this file at all, complaining that it could not recognize the characters (just those that differ between the original file and the powershell converted one). So far, I have rolled back the contents of the file to the original one. I will try it later in Ubuntu.